oauth2-server/pom.xml

```diff
@@ -4,11 +4,11 @@
   <parent>
     <groupId>com.clouway.security</groupId>
     <artifactId>parent</artifactId>
-    <version>1.0.6-SNAPSHOT</version>
+    <version>1.0.7-SNAPSHOT</version>
```

Comment:
Dumping happens automatically and this change is not required.

Reply:
http://maven.apache.org/maven-release/maven-release-plugin/

Reply:
ok I'll roll it back